### PR TITLE
Alexa Change Target URL

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
Ready

## Related Issues
fixes: link to the issue

## Description
Was pointing to http:// endpoint, is now pointing to https:// endpoint.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by amshamah419** [comment]
For some strange reason https:// is triggering secrets detection.

**Comment by amshamah419** [comment]
Didn't even think to whitelist the domain. Ill get it in there.

**Comment by yaakovi** [comment]
you can remove the disable-secret comments now.

File: Tests/secrets_white_list.json [hunk 1]
```diff
@@ -405,7 +405,7 @@
       "https://s3.us-west-2.amazonaws.com",
       "https://api.abuseipdb.com",
       "https://www.abuseipdb.com",
-      "http://data.alexa.com",
+      "https://data.alexa.com",
```
Comment (yaakovi) [inline Tests/secrets_white_list.json]:
```suggestion
      "data.alexa.com",
```

**Comment by glicht** [comment]
see:  https://github.com/demisto/content/blob/ef728a6a04e3b9d659c5ea9e8966e44fa37d55d7/Integrations/Alexa/Alexa.py#L22

**Comment by amshamah419** [comment]
I saw that. I'm pushing a new commit with all of it included

**Pull request merged**